### PR TITLE
QA-257: update CD pipeline to use the sync:image job from templates

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: none

Signed-off-by: Fabio Tranchitella <fabio.tranchitella@northern.tech>

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -14,6 +14,8 @@ include:
     file: '.gitlab-ci-check-docker-build.yml'
   - project: 'Northern.tech/Mender/mendertesting'
     file: '.gitlab-ci-github-status-updates.yml'
+  - project: 'Northern.tech/Mender/mendertesting'
+    file: '.gitlab-ci-check-docker-deploy.yml'
 
 # Function ported from template gitlab-ci-check-docker-build
 .export_docker_vars: &export_docker_vars |
```
Comment (lluiscampos) [inline .gitlab-ci.yml:21]: We don't need this any more.

**Review by lluiscampos — APPROVED** [review]
Thanks! :top:

File: .gitlab-ci.yml (hunk 1 above)
Comment (tranchitella) [reply .gitlab-ci.yml:21]: sorry @lluiscampos, which thing we don't need anymore?

Comment (tranchitella) [reply .gitlab-ci.yml]: understood, the `export_docker_vars`

**Pull request merged**